Formula/t/thors-serializer.rb

```diff
@@ -66,9 +66,9 @@ def install
           return 0;
       }
     EOS
-    system ENV.cxx, "-std=c++17", "test.cpp", "-o", "test",
+    system ENV.cxx, "-std=c++20", "test.cpp", "-o", "test",
            "-I#{Formula["boost"].opt_include}",
-           "-I#{include}", "-L#{lib}", "-lThorSerialize17", "-lThorsLogging17", "-ldl"
+           "-I#{include}", "-L#{lib}", "-lThorSerialize20", "-lThorsLogging20", "-ldl"
```

Comment:
why not?

```suggestion
           "-I#{include}", "-L#{lib}", "-lThorSerialize", "-lThorsLogging", "-ldl"
```

Reply:
I was trying to be specific.

I have no issue making the change (if you want it).
But I think being specific may catch errors. It made me runt the `brew test thors-serializer` :-)
I initially missed it but the test failed and I had to fix it.

I update the version every couple of years when the C++ standard becomes stable/standard.

Reply:
it would be the same deal, that is symlink to the specific version of the dylib, it would just help reduce the code change though.

Reply:
Fixed